packages/frontend/src/pages/ProjectSettings.tsx

```diff
@@ -91,6 +93,10 @@ const ProjectSettings: FC = () => {
                         title="Project Access"
                     />
                     <Tab id={IntegrationsTabs.DBT_CLOUD} title="dbt Cloud" />
+                    <Tab
```

Comment:
Not sure this tab should be visible for non-admins. Otherwise they have this option just to get a forbidden panel in the user analytics page.